node/variables.tf

```diff
@@ -218,3 +242,19 @@ variable "ff_wait_ready" {
   type    = bool
   default = false
 }
+
+variable "ff_with_kubeproxy" {
+  type = bool
+}
+
+variable "server_node_labels" {
+  type = map(string)
+  default = {
+    "node.kubernetes.io/exclude-from-external-load-balancers" = "true"
```

Comment:
no need for a default here, if it is already in the parent module (only increase confusion)